ac-core/src/main/java/es/ucm/fdi/ac/test/TokenCountTest.java

```diff
@@ -64,7 +64,7 @@ public void preprocess(Submission s) {
 		String tokens = getTokens(s);
 
 		TreeMap<Integer, Integer> counter = new TreeMap<Integer, Integer>();
-		StringTokenizer st = new StringTokenizer(tokens, "\n\r");
+		StringTokenizer st = new StringTokenizer(tokens, " ");
```

Comment:
typical antlr4 tokenized strings look like `3a 13 9 36 1r 3b 1s -1` (for the `sample.java` file included in ac-lexers test), so yes, this works. The old tokenizer (from PMD) generated output in columns. I would, however, prefer to use more whitespace characters to future-proof things somewhat: " \n\r\t"